crates/sc-piece-cache/src/lib.rs

```diff
@@ -76,6 +102,21 @@ where
             .to_multihash()
             .to_bytes()
     }
+
+    #[inline]
+    fn segments_number_in_cache(size: u64) -> u64 {
+        size / (PIECES_IN_SEGMENT as u64 * PIECE_SIZE as u64)
+    }
+
+    #[inline]
+    fn min_segments_number_in_cache() -> u64 {
+        Self::segments_number_in_cache(ONE_GB)
+    }
```

Comment:
Why do we have a minimum in the library here? It feels like library should not have such a notion.

Reply:
I would keep it as a safety mechanism.